uitest/Configurations/AppConfiguration.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft Corporation and Contributors
+// Licensed under the MIT license.
+
+namespace DevHome.UITest.Configurations;
+
+/// <summary>
+/// Class model of the appsettings JSON file root object
+/// </summary>
+public class AppConfiguration
+{
+    /// <summary>
+    /// Gets or sets the target Dev Home package family name
+    /// </summary>
+    public string PackageFamilyName { get; set; }
+
+    /// <summary>
+    /// Gets or sets the windows application driver url
+    /// </summary>
+    public string WindowsApplicationDriverUrl { get; set; }
```

Comment:
What is this? Is this another Appium thing I don't know about?

Reply:
This is the Windows Application driver url which we connect to from DevHomeSession (e.g. `http://127.0.0.1:4723/`)

```ps
PS> &"C:\Program Files\Windows Application Driver\WinAppDriver.exe"
Windows Application Driver listening for requests at: http://127.0.0.1:4723/
Press ENTER to exit.
```